### PR TITLE
Upgrade to the latest solidus_dev_support

### PR DESCRIPTION
**Pull request by vassalloandrea**

### CODE REVIEW

File: lib/solidus_comments.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
...
 
-require 'spree_api'
-require 'spree_backend'
 require 'solidus_core'
 require 'solidus_support'
 require 'acts_as_commentable'
+
+require 'solidus_comments/version'
 require 'solidus_comments/engine'
-require 'deface'
```
Comment (aldesantis) [inline lib/solidus_comments.rb]: If the extension has overrides, which it does, you need to require Deface for the overrides to work.

Comment (vassalloandrea) [reply lib/solidus_comments.rb]: I checked if deface works using the dummy app and it does. WDYT?

Comment (aldesantis) [reply lib/solidus_comments.rb]: Interesting... I've had this problem in the past, where removing `require 'deface'` would not display the overrides. After all, if you're not requiring Deface and no one else is, the gem cannot be loaded. Could you check if maybe something else is loading Deface in your tests?

Comment (vassalloandrea) [reply lib/solidus_comments.rb]: image

It seems that anyone require it. 
Maybe we can require it avoiding any trouble...

Comment (aldesantis) [reply lib/solidus_comments.rb]: Yeah, let's do that, because otherwise it's not clear how the overrides would be loaded.

Comment (kennyadsl) [reply lib/solidus_comments.rb]: could it be required by another extension? Maybe `solidus_auth_devise`?

**Pull request merged**